bin/coffeelint

```diff
@@ -3,11 +3,12 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 
 require 'cc/engine/coffeelint'
 
-def parse_engine_config(raw)
-  raw ? JSON.parse(raw) : {}
+if File.exists?("/config.json")
```

Comment:
@ABaldwinHunter We mount the json file ourselves when we make the engine container, and we mount it to `/config.json` so that it doesn't conflict with anything in the `/code` directory.

Spec change here (might help clarify): https://github.com/codeclimate/spec/pull/1